src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -245,14 +245,22 @@ public void ReplacePriceHistory(Party party, List<PriceHistory> currentPriceHist
             this.PriceHistory = updatedPriceHistory;
         }
 
-        public void UpdateCourse(Party party, string courseCode, string courseName, ProgrammeType programmeType, UserInfo userInfo)
+        public void UpdateCourse(Party party, string courseCode, string courseName, ProgrammeType programmeType, UserInfo userInfo, string standardUId)
         {
             StartTrackingSession(UserAction.UpdateCourse, party, Cohort.EmployerAccountId, Cohort.ProviderId, userInfo);
             ChangeTrackingSession.TrackUpdate(this);
 
             CourseCode = courseCode;
             CourseName = courseName;
             ProgrammeType = programmeType;
+            StandardUId = standardUId;
+
+            Publish(() =>
+            new ApprenticeshipUpdatedApprovedEvent
+            {
+                ApprenticeshipId = Id,
+                StandardUId = standardUId,
+            });
```

Comment:
This is a slight worry, I would assume if we publish an event, it would have all it's properties updated. We cannot assume other systems are not utilising these values. I think in CMAD we only use the ApprenticeshipId, so it should work for us.

Reply:
I've since added as much to the publish event data as can yield in this method